### ISSUE TITLE
No Ssr support after recent updates

### ISSUE THREAD

**Issue by vitaliizdrokapiko** [gm0t/react-sticky-el#59]
"window" is not available during server side rendering. With latest version, before it was ok

**Comment by yocmen** [comment]
yes, I think is for the new default prop of scrollelement = window, so I think this could be solved with something like:

```
    scrollElement: typeof window !== "undefined" ? window : null
``` 

Lets wait form @gm0t  and see what he think.

**Comment by gm0t** [comment]
Fixed in 2.0.3
